functorch/setup.py

```diff
@@ -1,149 +0,0 @@
-# Copyright (c) Facebook, Inc. and its affiliates.
-# All rights reserved.
-#
```

Comment:
We should have a dummy functorch/setup.py that is a one-liner that raises a warning that one does not need to use it anymore to smooth the transition. PyTorch devs will cd into functorch and attempt to run setup.py develop.

Also, it might be good to mention that the previous functorch installation needs to get uninstalled (I discovered this while running through https://docs.google.com/document/d/1nMiUkuewG0RfRMs2gNxr5VKBDlCm8xAgF8FrRLFw4lU/edit# )